hack/docker.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# This script holds docker related functions.
+
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+REGISTRY=${REGISTRY:-"swr.ap-southeast-1.myhuaweicloud.com/karmada"}
+VERSION=${VERSION:="unknown"}
```

Comment:
Can you (maybe in a separate PR) make default version to: latest tag + git_hash?

ref: https://github.com/kubernetes/kubernetes/blob/master/hack/lib/version.sh

Reply:
Sure.